SpatialGDK/Source/Private/Utils/ComponentReader.cpp

```diff
@@ -295,7 +297,14 @@ void ComponentReader::ApplyProperty(Schema_Object* Object, Schema_FieldId FieldI
 			if (NetGUID.IsValid())
 			{
 				UObject* ObjectValue = PackageMap->GetObjectFromNetGUID(NetGUID, true);
-				checkf(ObjectValue, TEXT("An object ref %s should map to a valid object."), *ObjectRef.ToString());
+				if (!ObjectValue)
+				{
+					// At this point, we're unable to resolve a stably-named actor by path. This likely means either the actor doesn't exist, or
+					// it's part of a streaming level that hasn't been streamed in. In either case, there's nothing we can do.
+					UE_LOG(LogSpatialComponentReader, Warning, TEXT("Object ref did not map to valid object, will be set to nullptr: %s %s"),
```

Comment:
Won't the `Path` here only contain the last segment of the path? E.g. for `/Game/Package.Thing`, this will only be `Thing`.

Reply:
That is a very good point. Will add a helper to print the full path.